upgrade/upd-2.0.18-to-2.3.0/index.php

```
@@ -280,24 +282,24 @@ public function update_configs($task)
-        while (false !== (list($table) = $GLOBALS['xoopsDB']->fetchRow($result))) {
+        while (false !== (list($table) = $this->db->fetchRow($result))) {
             $tables[] = $table;
-            //$GLOBALS["xoopsDB"]->exec( "ALTER TABLE `{$table}` DEFAULT CHARACTER SET " . $GLOBALS["xoopsDB"]->quote($charset) . " COLLATE " . $GLOBALS["xoopsDB"]->quote($collation) );
-            //$GLOBALS["xoopsDB"]->exec( "ALTER TABLE `{$table}` CONVERT TO CHARACTER SET " . $GLOBALS["xoopsDB"]->quote($charset) . " COLLATE " . $GLOBALS["xoopsDB"]->quote($collation) );
+            //$this->db->exec( "ALTER TABLE `{$table}` DEFAULT CHARACTER SET " . $this->db->quote($charset) . " COLLATE " . $this->db->quote($collation) );
+            //$this->db->exec( "ALTER TABLE `{$table}` CONVERT TO CHARACTER SET " . $this->db->quote($charset) . " COLLATE " . $this->db->quote($collation) );
         }
         $this->convert_table($tables, $charset, $collation);
-        return null;
+        return true;
     }
```

PHPMD (2.15.0)

[error] The method convert_db is not named in camelCase. (undefined)

(CamelCaseMethodName)